geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/SimpleInstancesExample.js

```diff
@@ -66,8 +66,8 @@ class SimpleInstancesExample extends Component {
         reset: false,
         autorotate: false,
         wireframe: false,
-        showModel: false
       },
+      showModel: false
```

Comment:
is this change related to the widget issue?

Reply:
No